atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolTest.java

```diff
@@ -41,7 +41,7 @@
 import com.palantir.common.base.FunctionCheckedException;
 
 public class CassandraClientPoolTest {
-    public static final int POOL_REFRESH_INTERVAL_SECONDS = 10;
+    public static final int POOL_REFRESH_INTERVAL_SECONDS = 3 * 60;
```

Comment:
This seems like an extremely long time to wait for tests, although the most recent test run doesn't seem to take egregiously long. Do we ever actually hit this refresh?